schemas/publicationIssue.schema.tpl.json

```diff
@@ -0,0 +1,16 @@
+{
+  "_type": "https://openminds.ebrains.eu/publications/PublicationIssue",
+  "required": ["isPartOf", "issueNumber"],
+  "properties": {
+    "issueNumber": {
+      "type": "string",
+      "_instruction": "Enter the number of this issue."
+    },
+    "isPartOf": {
+      "_instruction": "Add the volume this issue is part of.",
+      "_embeddedTypes": [
+        "https://openminds.ebrains.eu/publications/PublicationVolume"
```

Comment:
should be linked type since multiple issues can be part of the same volume

Reply:
Having them linked would make sense if we cared whether two articles belonged to the same issue, but in practice we don't care. Having them embedded makes code simpler and queries potentially faster, since we don't have to follow the links in the graph.

An alternative, as I mentioned above, would be to remove these schemas, and just have string fields "volume" and "issue" in Periodical.

Reply:
the latter alternative I do not like because we would need to register journal information over and over again. If you want to register them directly and get rid of the schemas you should integrate them into the ScholarlyArticle schema as properties. that would be okay with me as alternative.

IMO we should stick to graph structures wherever we can and only consider code simplification where no joint information between instances is to be expected. Meaning I would make them linked types and live with the resulting query structure.